test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/SystemDataSqlClientTests.cs

```diff
@@ -42,19 +42,20 @@ public void SubmitsTracesWithNetStandard(string packageVersion, bool enableCallT
             // Once this is fully supported, this will add another 2 complete groups for all frameworks instead
             // of 4 extra spans on net461 and netcoreapp2.0+
 #if NET452
-            var expectedSpanCount = 70; // 7 queries * 10 groups
-#elif NET461
             var expectedSpanCount = 98; // 7 queries * 14 groups
+#elif NET461
+            var expectedSpanCount = 154; // 7 queries * 14 groups
 #else
-            var expectedSpanCount = 102; // 7 queries * 14 groups + 4 spans from generic wrapper on .NET Core
+            // Update to final value once tests pass
+            var expectedSpanCount = 158; // 7 queries * 14 groups + 4 spans from generic wrapper on .NET Core
```

Comment:
As above RE comments

Reply:
I put off updating these 😅 Done in f9829db87629aa5c357cc4ad5a69a51fcd1da1ad